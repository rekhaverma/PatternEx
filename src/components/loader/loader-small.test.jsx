import React from 'react';
import LoaderSmall from './loader-small.component';

/**
 * @status: Complete
 * @sign-off-by: Alex Andries
 */
describe('<LoaderSmall />', () => {
  it('should match with snapshot', () => {
    const wrapper = shallow(<LoaderSmall />);
    expect(wrapper).toMatchSnapshot();
  });
});
