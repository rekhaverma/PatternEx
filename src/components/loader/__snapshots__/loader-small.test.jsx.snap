// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoaderSmall /> should match with snapshot 1`] = `
<div
  className="loaderSmall"
>
  <svg
    className="loaderSmall__circular"
    viewBox="25 25 50 50"
  >
    <circle
      className="loaderSmall__path"
      cx="50"
      cy="50"
      fill="none"
      r="20"
      strokeMiterlimit="10"
      strokeWidth="3"
    />
  </svg>
</div>
`;
